### ISSUE TITLE
Remove deprecated ``astroid.node_classes`` and ``astroid.scoped_nodes``

### ISSUE THREAD

**Issue by Pierre-Sassoulas** [pylint-dev/astroid#1072]
Those two files have been deprecated in 2.7.0 and need to be removed in 3.0.0.

**Comment by belm0** [comment]
astroid seems to be triggering its own deprecation warnings, which is quite annoying

https://github.com/PyCQA/astroid/blob/4ffdf1108a6084b85e282b5835427c1d747bb22c/astroid/__init__.py#L54-L55

```
  File "/.../site-packages/astroid/__init__.py", line 54, in <module>
    from astroid import node_classes  # Deprecated, to remove later
  File "/.../site-packages/astroid/node_classes.py", line 90, in <module>
    warnings.warn(
DeprecationWarning: The 'astroid.node_classes' module is deprecated and will be replaced by 'astroid.nodes' in astroid 3.0.0
```

**Comment by Pierre-Sassoulas** [comment]
@belm0 thank you for the coment, I opened #1190 to fix that, should be available in 2.8.1.
